src/main/java/org/concordion/Concordion.java

```diff
@@ -78,7 +78,7 @@ public void override(Resource resource) throws IOException {
     public ResultSummary process(Fixture fixture) throws IOException {
         SummarizingResultRecorder resultRecorder = new SummarizingResultRecorder();
         resultRecorder.setSpecificationDescription(fixture.getSpecificationDescription());
-        getSpecification(fixture).process(evaluatorFactory.createEvaluator(fixture.getFixtureObject()), resultRecorder);
+        getSpecification(fixture).process(evaluatorFactory.createEvaluator(fixture.getFixtureObject()), resultRecorder, fixture);
```

Comment:
It seems odd having to pass the fixture to two calls here on the same line.

Reply:
Good spot, SpecificationByExample holds the fixture, so I'll remove it from the `process` method.

Reply:
I've modified the handling so `getSpecification()` no longer requires a fixture